src/index.js

```diff
@@ -15,48 +13,3 @@ ReactDOM.render (
       <App/>
   </Provider>, document.getElementById('root')
 )
-
-// serviceWorker.register()
-
-// export default class AppRouter extends Component {
-//   constructor(props) {
-//     super(props)
-//     this.store = createStore(rootReducers,applyMiddleware(thunk))
-//   }
-
-//   render() {
-//     console.log("hello")
-//     return (
-//       <Provider store={store}>
-//           {/* <ConnectedRouter history={history} > */}
-//             <Switch>
-//               <Route path='/' component={App} />
-//             </Switch>
-//           {/* </ConnectedRouter> */}
-//       </Provider>, document.getElementById('root')
-//     )
-//   }
-// }
-
-
-  
-
-// export default class AppRouter extends Component{
-//   render() {
-//     return(
-//       <Provider store={store}>
-//         <ConnectedRouter history={history} >
-//             <Switch>
-//               <Route path='/' component={App} />
-//             </Switch>
-//           </ConnectedRouter>
-//       </Provider>
-//     )
-//   }
-// }
-
-
-
-// If you want your app to work offline and load faster, you can change
-// unregister() to register() below. Note this comes with some pitfalls.
-// Learn more about service workers: https://bit.ly/CRA-PWA
```

Comment:
Doesn't belong here.

Reply:
These were unnecessary comments which I had added, so I removed them!

Reply:
True yes, it does make sense to make this change - but can you shift this to a different PR? This does not belong to this issue, nor does it belong to any other issues you made this change in. I'll create an issue for it, if it helps?